frontend/js/i18n/tr-lang.json

```
@@ -0,0 +1,338 @@
+		"fancy-index": "Fancy Index",
+		"fancy-index-help": "Dosya listelerken index kullanmak için",
+		"generate": "Oluştur",
+		"user-config": "Kullanıcı Yapılandırması",
+		"title": "Ayarlar"
+	},
+	"token": {
+		"add": "API Anahtarı Ekle",
+		"copy": "Kopyala",
+		"delete": "API Anahtarını Sil",
+		"delete-confirm": "Bu API Anahtarını silmek istediğinizden emin misiniz?",
+		"edit": "API Anahtarını Düzenle",
+		"empty": "API Anahtarı bulunmuyor",
+		"expires-on": "Süresi bitiyor: {date}",
+		"form-title": "{id, select, undefined{Yeni} other{Düzenle}} API Anahtarı",
+		"help-content": "API anahtarları, NPMplus API'sine harici uygulamalar üzerinden erişim sağlamak için kullanılır.",
+		"help-title": "API Anahtarı Nedir?",
+		"scope": "Kapsam",
+		"search": "Anahtar Ara…",
+		"title": "API Anahtarları",
+		"use-expiry": "Süre Sonu Kullan"
+	},
+	"var": {
+		"and": "ve",
+		"and-other": "ve {count} diğer",
+		"cancel": "İptal",
+		"choose-file": "Dosya seç",
+		"close": "Kapat",
+		"created-on": "Oluşturulma: {date}",
+		"delete": "Sil",
+		"destination": "Hedef",
+		"disable": "Devre Dışı Bırak",
+		"disabled": "Devre Dışı",
+		"edit": "Düzenle",
+		"email": "E-posta",
+		"email-address": "E-posta adresi",
+		"enable": "Etkinleştir",
+		"expires": "Süresi Doluyor",
+		"name": "Ad",
+		"offline": "Çevrimdışı",
+		"online": "Çevrimiçi",
+		"password": "Şifre",
+		"please-wait": "Lütfen bekleyin...",
+		"public": "Herkese Açık",
+		"roles": "Roller",
+		"save": "Kaydet",
+		"sign-in": "Giriş yap",
+		"sign-in-with": "İle giriş yap",
+		"sign-out": "Çıkış yap",
+		"source": "Kaynak",
+		"status": "Durum",
+		"sure": "Evet Eminim",
+		"tls": "TLS",
+		"try-again": "Tekrar dene",
+		"unknown": "Bilinmiyor",
+		"username": "Kullanıcı Adı",
+		"value": "Değer"
+	},
+	"streams": {
+		"add": "Akış Ekle",
+		"delete": "Akışı Sil",
+		"delete-confirm": "Bu Akışı silmek istediğinizden emin misiniz?",
+		"empty": "Akış bulunmuyor",
+		"form-title": "{id, select, undefined{Yeni} other{Düzenle}} Akış",
+		"forward-type-error": "En az bir protokol türü etkinleştirilmelidir",
+		"forwarding-host": "Yönlendirme Sunucusu",
+		"forwarding-port": "Yönlendirme Portu",
+		"help-content": "Nginx için nispeten yeni bir özellik olan Akış, TCP/UDP trafiğini doğrudan ağdaki başka bir bilgisayara yönlendirmeye yarar.\nOyun sunucuları, FTP veya SSH sunucuları çalıştırıyorsanız bu işe yarayabilir.",
+		"help-title": "Akış Nedir?",
+		"incoming-port": "Gelen Port",
+		"protocol": "Protokol",
+		"proxy-protocol-forwarding": "Upstream proxy protokolü kullanır",
+		"search": "Akış Ara…",
+		"tls-certificate": "TCP Yönlendirme için TLS Sertifikası",
+		"tcp": "TCP",
+		"tcp+pp": "TCP+proxy protokolü",
+		"tcp+tls": "TCP+TLS",
+		"tcp+pp+tls": "TCP+proxy protokolü+TLS",
+		"tcp-forwarding": "TCP Yönlendirme",
+		"title": "Akışlar",
+		"udp": "UDP",
+		"udp+pp": "UDP+proxy protokolü",
+		"udp+tls": "UDP+TLS",
+		"udp+pp+tls": "UDP+proxy protokolü+TLS",
```

```suggestion
		"tcp+pp": "TCP+proxy protocol",
		"tcp+tls": "TCP+TLS",
		"tcp+pp+tls": "TCP+proxy protocol+TLS",
		"tcp-forwarding": "TCP Yönlendirme",
		"title": "Akışlar",
		"udp": "UDP",
		"udp+pp": "UDP+proxy protocol",
		"udp+tls": "UDP+TLS",
		"udp+pp+tls": "UDP+proxy protocol+TLS",
```
should not be translated